openlibrary/tests/accounts/test_models.py

```diff
@@ -0,0 +1,7 @@
+from openlibrary.accounts import model
+
+def test_verify_hash():
+      secret_key = "aqXwLJVOcV"
+      hash = model.generate_hash(secret_key, "foo")
+      assert model.verify_hash(secret_key, "foo", hash) == True
```

Comment:
we should be doing a comparison against True using `is` rather than ==
I understand this is a pre-existing piece of code.
Otherwise, looks good to me -- I'll merge.